pkgs/by-name/ll/llama-cpp/package.nix

```diff
@@ -150,7 +153,7 @@ effectiveStdenv.mkDerivation (finalAttrs: {
     homepage = "https://github.com/ggerganov/llama.cpp/";
     license = licenses.mit;
     mainProgram = "llama";
-    maintainers = with maintainers; [ dit7ya elohmeier philiptaron ];
+    maintainers = with maintainers; [ dit7ya elohmeier philiptaron ck3d ];
```

Comment:
why do you want to be a maintainer? this is a pretty active packages but this looks like your first commit to the package.

Reply:
1. I packaged local-ai, which depends on llama-cpp.
2. I saw breaking changes in the past, so I wanted to help.

Never mind, I removed myself from this list.

Reply:
IMHO, having new maintainers is fine.

Reply:
> IMHO, having new maintainers is fine.

I agree.